### PR TITLE
my questions Anar Abetayeva C04

### PR DESCRIPTION
**Pull request by 4nar**

### CODE REVIEW

**Comment by konsteacher** [comment]
Remove *.~ files

**Comment by 4nar** [comment]
ok. just submit it in one commit?

**Comment by 4nar** [comment]
teacher, I just got it )) ok, i will remove them

File: git/question_r6F4D.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+What is git?
+* web site
+* repository
++ version control system
+* general intelligence test
+* software
```
Comment (konsteacher) [inline git/question_r6F4D.md]: is it not a software?

File: linux/question_bq4Rb.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+Change the date on yesterday.txt to match yesterday's date.
++ touch -t 201512281145 yesterday.txt
+* mv -t 201512281145 yesterday.txt
+* cp -t 201512281145 yesterday.txt
+* cat -t 201512281145 yesterday.txt
```
Comment (konsteacher) [inline linux/question_bq4Rb.md]: Smart)

**Pull request merged**